pulsar-broker/src/main/java/org/apache/pulsar/broker/service/BrokerService.java

```diff
@@ -1027,14 +1006,7 @@ public void unloadNamespaceBundlesGracefully(int maxConcurrentUnload, boolean cl
                         new IllegalStateException("Delete forbidden topic is replicated on clusters " + clusters));
             }
 
-            if (deleteSchema) {
-                return t.deleteSchema().thenCompose(schemaVersion -> {
-                    log.info("Successfully delete topic {}'s schema of version {}", t.getName(), schemaVersion);
-                    return t.delete();
-                });
```

Comment:
If remove these lines how do we delete the schema?

Reply:
L1009 will delete schema but default.